data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2017 by Michael Zahniser
```

Comment:
For these new files you're adding, probably the copyright is yours, or at least in addition to Michael's.

Reply:
I created the file, but any number of changes could be added in the future such that the work I originally added is only a minority of the file, so I wouldn't really feel justified in attaching my name to it. The fact that I made the file is recorded on Github anyway.